Sources/SwiftDocC/Infrastructure/Workspace/DefaultAvailability.swift

```diff
@@ -26,6 +28,12 @@ import Foundation
 ///             <key>version</key>
 ///             <string>Version Number</string>
 ///         </dict>
+///         <dict>
+///             <key>name</key>
+///             <string>Platform Name</string>
```

Comment:
minor: I don't think we support specifying the same platform more than once

```suggestion
///             <string>Other Platform Name</string>
```

Reply:
Resolved https://github.com/apple/swift-docc/pull/807/commits/768efb21f647046dd13622f1658062f931ef9bd0